charts/enterprise/clusterissuer/templates/clusterissuer/_clusterCertificates.tpl

```diff
@@ -0,0 +1,30 @@
+{{- define "certmanager.clusterissuer.clusterCertificates" -}}
+{{- if .Values.clusterCertificates -}}
+  {{- $certs := dict }}
+  {{- $secretTemplates := dict }}
```

Comment:
```suggestion
  {{- $secretTemplates := dict -}}
```

Add `-` everywhere here, as you dont "print" anything.
Lets not output empty space!

Reply:
Didn't know that is the the difference! Will keep it in mind for the future. Thanks